docs/source/installation.rst

```diff
@@ -31,7 +31,7 @@ can type
 We have binary packages for OSX, Linux and Windows (64 bit only) available from
 `PyPI <https://pypi.org/project/pyopenms>`_. Make sure to download
 the 64bit Python release for Windows. Currently we only support
-Python 3.7, 3.8 and 3.9.
+Python 3.7, 3.8 and 3.9, if you are using 3.10 or 3.11 please use our nightly builds.
```

Comment:
I think it would be better to have versioned docs. I.e. a nice visible drop-down on the web interface such that people can switch from nightly to release.

Reply:
It is possible by navigating the RTD inlet on the lower left, but not very visible.

Reply:
See https://stackoverflow.com/questions/49331914/enable-versions-in-sidebar-in-sphinx-read-the-docs-theme

how to display versioned docs better. Feel free to delegate to the doc team.

Reply:
Okay. I'm going to open that as a new issue, have the docs folks handle it and merge in these changes, as they still mark an improvement.